src/Installers/Windows/WindowsHostingBundle/Bundle.wxs

```diff
@@ -13,6 +13,11 @@
             <PayloadGroupRef Id="PG_Resources"/>
         </BootstrapperApplicationRef>
 
+        <!-- Search references for upgrade policy keys -->
```

Comment:
You'll need to add these to the ASP.NET bundle as well

Reply:
Done